perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -889,8 +889,9 @@ sub process_report : Private {
             if $body_string && @{ $c->stash->{missing_details_bodies} };
         $report->bodies_str($body_string);
 
-        my @extra = ();
-        my $metas = $contacts[0]->extra;
+        my @extra;
+        # NB: we are only checking extras for the *first* retrieved contact.
+        my $metas = $contacts[0]->get_extra_fields($c);
```

Comment:
contact `extra` is also looked up on line 652 (for use by the `category_extras` ajax).